Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml

```
@@ -1622,6 +1712,6 @@ inputs:
   description: Attack Surface Management Rule ID.
   playbookInputQuery:
 outputs: []
-fromversion: 6.5.0
+fromversion: 6.9.0
```

```suggestion
fromversion: 6.5.0
```
fix validation error

---

resolved